src/app/governance/page.tsx

```diff
@@ -185,25 +189,24 @@ export default function Governance() {
         </article>
       </section>
 
-      <section>
-        <Heading tag="h2" variant="xl">
-          Get Involved
-        </Heading>
-        <Social />
-        <p>Community governance calls happen every other Tuesday.</p>
-        <p>
-          Visit the{' '}
-          <TextLink href={PATHS.EVENTS.path}>
-            Filecoin community events hub
-          </TextLink>{' '}
-          and join the conversation in the{' '}
-          <TextLink href={FIL_PLUS_URLS.slack}>
-            #fil-plus Slack channel
-          </TextLink>
-          .
-        </p>
-        <TextLink href={PATHS.GET_INVOLVED.path}>Get Involved</TextLink>
-      </section>
+      <CTASection
+        title="Connect With Us"
+        description={
+          <>
+            To get involved, visit the{' '}
+            <TextLink href={FIL_PLUS_URLS.notary}>Governance GitHub</TextLink>,
+            join the conversation in the{' '}
+            <TextLink href={FILECOIN_URLS.social.slack.href}>
+              #fil-gov Slack channel
+            </TextLink>
+            , or follow{' '}
+            <TextLink href={FILECOIN_FOUNDATION_URLS.social.twitter.href}>
+              @fil_gov
+            </TextLink>{' '}
+            on X.
```

Comment:
I don't think any of these are the right links.

Reply:
Let's create a nested governance > social under FILECOIN_FOUNDATION_URLS` with the right data.

Reply:
^^ Already started this in other PR.